README.md

```diff
@@ -134,18 +155,14 @@ With snap, in any of the [supported Linux distributions](https://snapcraft.io/do
 $ sudo snap install ipfs
 ```
 
-### From Windows package managers
+### Windows package managers
 
 - [Chocolatey](#chocolatey)
 - [Scoop](#scoop)
 
 #### Chocolatey
 
-The package [ipfs](https://chocolatey.org/packages/ipfs) currently points to go-ipfs and is being maintained.
-
-```Powershell
-PS> choco install ipfs
-```
+[![Chocolatey Version](https://img.shields.io/chocolatey/v/go-ipfs?color=00a4ef&label=go-ipfs&logo=windows&style=for-the-badge&cacheSeconds=3600)](https://chocolatey.org/packages/go-ipfs)
```

Comment:
There's no reason to remove the instructions.

Reply:
Well, you can visit the site and copy the instruction from it. I thought that's favorable for users :)

Reply:
It also has the advantage to show the disclaimer for package security and provide further help. I think that's better than showing a random command string which feels a bit like `curl | sh`

Reply:
Please put it back. It's nothing like `curl | sh`, it's a nice bit of documentation like every other install command included in the readme.

Reply:
okay